tests/connect/standalone-plain-0001.phpt

```diff
@@ -1,8 +1,8 @@
 --TEST--
 Connect to MongoDB with using PLAIN auth mechanism
 --SKIPIF--
+<?php echo "skip authMechanism=PLAIN (LDAP) tests must be reimplemented (PHPC-1172)\n"; ?>
```

Comment:
I prefer having these as XFAIL for now, and not just skipped.

Reply:
Will make the change across the board for these.